pkgs/development/python-modules/pykalman/default.nix

```diff
@@ -29,6 +29,11 @@ buildPythonPackage rec {
     scikit-base
   ];
 
+  postPatch = ''
+    substituteInPlace pyproject.toml \
+      --replace-fail 'scikit-base<0.13.0' 'scikit-base>=0.12.0'
+  '';
+
```

Comment:
Have you tried using `pythonRelaxDeps` instead?

Reply:
yes tried

Reply:
You have to put `"skbase"`, not `"scikit-base"`. Not intuitive I'll admit.